src/daemon/dlt-daemon.c

```diff
@@ -456,6 +457,12 @@ int option_file_parser(DltDaemonLocal *daemon_local)
                         daemon_local->flags.sharedMemorySize = atoi(value);
                         /*printf("Option: %s=%s\n",token,value); */
                     }
+                    else if (strcmp(token, "OfflineTraceOn") == 0)
+                    {
+                        daemon_local->flags.offlineTraceOn = atoi(value);
+                        fprintf(stderr,"OfflineTraceOn==%d\n",daemon_local->flags.offlineTraceOn);
```

Comment:
I think it should "OfflineTraceOn=%d\n" instead of "OfflineTraceOn==%d\n"

Reply:
... and should be removed all togther -- debug code -- thanks for pointing out